wiki/Beginner/en.md

```diff
@@ -47,104 +49,99 @@ Touchscreen tends to struggle with:
+-   [fripSide - a little christmas time](https://osu.ppy.sh/b/191008) on Easy (1.03 Stars)
+-   [Yoko Shimomura - A Twinkle in the Sky](https://osu.ppy.sh/b/220650) on Easy (1.05 Stars)
+-   [Sean Paul - She Doesn't Mind (Daniel Ngo Remix)](https://osu.ppy.sh/b/158917) on Easy (1.08 Stars)
+-   [Anamanaguchi - This is the End](https://osu.ppy.sh/b/74686) on Average Joe (1.09 Stars)
+-   [Infected Mushroom - The Pretender](https://osu.ppy.sh/b/240979) on Easy (1.10 Stars)
+-   [Do As Infinity - Fukai Mori](https://osu.ppy.sh/b/49478) on Easy Dan (1.11 Stars)
+-   [Nishiura Tomohito - Shop](https://osu.ppy.sh/b/404645) on Easy (1.13 Stars)
+-   [Angela Chang - Aurora](https://osu.ppy.sh/b/397975) on Easy (1.14 Stars)
+
+### Novice
 
 After you play a bit, try these maps out.
 
--   [Mutsuhiko Izumi - Russian Snowy Dance](http://osu.ppy.sh/b/124348) on Easy (1.21 Stars)
--   [goreshit - o'er the flood](http://osu.ppy.sh/b/177139) on Easy (1.22 Stars)
--   [Hatsune Miku - Electric Love (Short Ver.)](http://osu.ppy.sh/b/141749) on Easy. (1.25 Stars)
--   [DJ YOSHITAKA feat. Hoshino Kanako - MAX LOVE](http://osu.ppy.sh/b/186601) on Easy (1.28 Stars)
--   [Akcent feat. Edward Maya - That's My Name](http://osu.ppy.sh/b/132838) on Easy (1.28 Stars)
--   [Hatsune Miku - Do-re-mi-fa Rondo](http://osu.ppy.sh/b/163457) on Easy (1.28 Stars)
+-   [Mutsuhiko Izumi - Russian Snowy Dance](https://osu.ppy.sh/b/124348) on Easy (1.21 Stars)
+-   [goreshit - o'er the flood](https://osu.ppy.sh/b/177139) on Easy (1.22 Stars)
+-   [Hatsune Miku - Electric Love (Short Ver.)](https://osu.ppy.sh/b/141749) on Easy. (1.25 Stars)
+-   [DJ YOSHITAKA feat. Hoshino Kanako - MAX LOVE](https://osu.ppy.sh/b/186601) on Easy (1.28 Stars)
+-   [Akcent feat. Edward Maya - That's My Name](https://osu.ppy.sh/b/132838) on Easy (1.28 Stars)
+-   [Hatsune Miku - Do-re-mi-fa Rondo](https://osu.ppy.sh/b/163457) on Easy (1.28 Stars)
 
-Challenge
----------
+### Challenge
 
 Want to push yourself and take on a challenge? Try these!
 
--   [DJ Fresh - Gold Dust](http://osu.ppy.sh/b/96042) on Easy (1.32 Stars)
--   [Kyary Pamyu Pamyu - Ninjya Re Bang Bang](http://osu.ppy.sh/b/230174) on Easy (1.36 Stars)
--   [Carly Rae Jepsen - Call Me Maybe (Nightcore Mix)](http://osu.ppy.sh/b/172736) on Easy (1.36 Stars)
--   [Kicks N Licks Feat. Nicole Millar - Own The Night (MitiS DnB Remix)](http://osu.ppy.sh/b/368307) on Easy (1.45 Stars)
--   [Gentle Stick X M2U - Hades in the Heaven](http://osu.ppy.sh/b/318994) on Easy (1.46 Stars)
+-   [DJ Fresh - Gold Dust](https://osu.ppy.sh/b/96042) on Easy (1.32 Stars)
+-   [Kyary Pamyu Pamyu - Ninjya Re Bang Bang](https://osu.ppy.sh/b/230174) on Easy (1.36 Stars)
+-   [Carly Rae Jepsen - Call Me Maybe (Nightcore Mix)](https://osu.ppy.sh/b/172736) on Easy (1.36 Stars)
+-   [Kicks N Licks Feat. Nicole Millar - Own The Night (MitiS DnB Remix)](https://osu.ppy.sh/b/368307) on Easy (1.45 Stars)
```

Comment:
use `Kicks N Licks Feat. Nicole Millar - Own The Night (MitiS DnB Remix) (Ami) \[Easy\]` as link text